src/components/common-components/LargeTextBlock/LargeTextBlock.js

```diff
@@ -0,0 +1,50 @@
+import React, {useState} from 'react';
```

Comment:
nitpick: use spaces when deconstructing like this

```suggestion
import React, { useState } from 'react';
```